src/manage_migrators.c

```diff
@@ -2391,6 +2391,40 @@ migrate_234_to_235 ()
   return 0;
 }
 
+/**
+ * @brief Migrate the database from version 235 to version 236.
+ *
+ * @return 0 success, -1 error.
+ */
+int
+migrate_235_to_236 ()
+{
+  sql_begin_immediate ();
+
+  /* Ensure that the database is currently version 235. */
+
+  if (manage_db_version () != 235)
+    {
+      sql_rollback ();
+      return -1;
+    }
+
+  /* Update the database. */
+
+  /* Replace any result type "Debug Message" by "Error Message". */
+
+  sql ("UPDATE results SET type = 'Error Message' WHERE type = 'Debug Message';");
```

Comment:
Technically I guess this should set severity to -3.0.  But realistically I don't think there are any debug messages out there.